schemas/dart_model.schema.json

```diff
@@ -8,14 +8,40 @@
   "$defs": {
     "Augmentation": {
       "type": "object",
-      "description": "An augmentation to Dart code. TODO(davidmorgan): this is a placeholder.",
+      "description": "An augmentation to Dart code.",
       "properties": {
         "code": {
-          "type": "string",
-          "description": "Augmentation code."
+          "type": "array",
+          "description": "Augmentation code.",
+          "items": {
+            "$ref": "#/$defs/Code"
+          }
         }
       }
     },
+    "Code": {
+      "type": "object",
+      "description": "Code that is part of augmentations to Dart code.",
+      "properties": {
+        "type": {
+          "type": "string"
+        },
+        "value": {
+          "oneOf": [
+            {
+              "$ref": "#/$defs/QualifiedName"
+            },
+            {
+              "$ref": "#/$defs/ResolvedCode"
```

Comment:
I don't really understand what "ResolvedCode" means. What is unresolved code?

Reply:
Do we really need both of these kinds of Code at all? Qualified names are effectively resolved right?

I am pretty confused by what exactly these two things together are accomplishing.

Reply:
Changed `ResolvedCode` to `String`, with a small tweak to the generator to make that work. Much better now, thanks :)